modules/imgproc/src/resize.cpp

```diff
@@ -3902,6 +3902,9 @@ void resize(int src_type,
         resize_bitExact<int, interpolationLinear<int> >,
         0,
         0,
+        0,
+        0,
+        resize_bitExact<uchar, interpolationLinear<uchar> >, /* resize bool as uchar */
```

Comment:
As I understand `CV_Bool` should have 255 as `true` value. How will resize work in this case? Wouldn't it create `true` values with other values than 255. Perhaps test would be useful to check this case.